cellprofiler/modules/imagemath.py

```diff
@@ -505,10 +515,9 @@ def validate_module(self, pipeline):
 
     def upgrade_settings(self, setting_values, variable_revision_number,
                          module_name, from_matlab):
-        if (from_matlab and module_name == 'Subtract' and
-                    variable_revision_number == 3):
+        if (from_matlab and module_name == 'Subtract' and variable_revision_number == 3):
             subtract_image_name, basic_image_name, resulting_image_name, \
-            multiply_factor_1, multiply_factor_2, truncate = setting_values
+                multiply_factor_1, multiply_factor_2, truncate = setting_values
```

Comment:
Why did this tab get thrown in here?

Reply:
It's the required indentation level for a continuing line (note the `\` ending the line above).

Reply:
AH okay, the file diff makes it look a little wonky.